app/models/surplu.rb

```diff
@@ -1,4 +1,5 @@
 class Surplu < ApplicationRecord
   validates :resource, presence: true
   belongs_to :organization
+  has_many :claims, dependent: :destroy
```

Comment:
Same here. since a claim also belongs to an organization perhaps it should not be destroyed if a surplus is destroyed.